packages/cli/src/constructs/alert-channel.ts

```diff
@@ -54,11 +66,15 @@ export abstract class AlertChannel extends Construct {
     this.sslExpiryThreshold = props.sslExpiryThreshold
   }
 
+  static fromId (id: string|number) {
+    return new AlertChannelWrapper(`alert-channel-${id}`, id)
```

Comment:
Nevermind, since this is the physical UUID, I don't think that users will run into a duplication. It might still be worth changing the formatting, though, IMO (as a nit).